### ISSUE TITLE
FIX speed button does not always respond

### ISSUE THREAD

**Issue by allanlaal** [SkyTubeTeam/SkyTube#1066]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**
If applicable, add screenshots to help explain your problem, or the link for the video, where You've found the issue

**Setup:**
 - Android version: [e.g. 7 or 10, etc]
 - Is Your device rooted?
 - Do You use Tor or any VPN service with your device?
 - App source [e.g. FDroid, Downloaded from GitHub, self build]
 - App Version [e.g. v2.977]
 - App Flavour [e.g. Extra or OSS]

**Additional context**

**Comment by dagofr01** [comment]
I agree. I have a plain vanilla Nokia 6.1 with A10. Same with Skytube and Skytube extra. I guess I see it with N6 and A9 also but I verify....
This is intermittent - no pattern when. But not rare - Probably more than 50% chance.
